apps/examples/protobuf/protobuf_msgtype_test/Make.defs

```diff
@@ -0,0 +1,21 @@
+###########################################################################
+#
+# Copyright 2018 Samsung Electronics All Rights Reserved.
```

Comment:
Please change to 2021.

Reply:
I just changed the location and name of directory so don'y need to update copyright I think.